### ISSUE TITLE
crashh if change from potrait to landscape or vice versa during sytem loading

### ISSUE THREAD

**Issue by muhammadarif1** [robertoissc/DailyPrayer-Android#22]
but if screen rotated after fully loaded is work as well. but before fully loaded this app crash itself

step reproduce
-open the app and before the app fully loaded change the view 

recording of the bug
https://youtu.be/kbVV7dNKkL4
logcat
```FATAL EXCEPTION: main
Process: com.isscroberto.dailyprayerandroid, PID: 10777
java.lang.NullPointerException: Attempt to invoke interface method 'void com.isscroberto.dailyprayerandroid.prayer.PrayerContract$View.showPrayer(com.isscroberto.dailyprayerandroid.data.models.Item)' on a null object reference
	at com.isscroberto.dailyprayerandroid.prayer.PrayerPresenter$1.onResponse(PrayerPresenter.java:66)
	at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run(ExecutorCallAdapterFactory.java:68)
	at android.os.Handler.handleCallback(Handler.java:815)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loop(Looper.java:218)
	at android.app.ActivityThread.main(ActivityThread.java:5657)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:990)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:785)```

**Comment by roberto-o-r** [comment]
Tested and confirmed, the fix will be included with next version. Thank you for the report!

**Comment by roberto-o-r** [comment]
Fixed on #14 👍

**Comment by muhammadarif1** [comment]
thank for your respond
